apis/hive/v1/metricsconfig/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package metricsconfig contains API Schema definitions for configurations specific to metrics controller.
+// +k8s:deepcopy-gen=package,register
+package metricsconfig
```

Comment:
The other packages' doc.go files also have
```suggestion
// +k8s:conversion-gen=github.com/openshift/hive/apis/hive
package metricsconfig
```
I actually have no idea what this does or why it's there. I can't get it to behave differently with `update-codegen` either. Any ideas?

(`conversion-gen` is documented [here](https://pkg.go.dev/k8s.io/code-generator/cmd/conversion-gen) -- but we don't seem to be using it.)

Reply:
Couldn't figure out why it's not present in every api module and why it'd be needed here, but added it anyways for future purposes when this module expands and that conversion would be needed - doesn't hurt us either way